report_templates/index.Rmd.in

```diff
@@ -1,45 +1,73 @@
 ---
-title: "PiGx: BS-seq"
-output: 
-  html_notebook: 
-    toc:        TRUE
-    toc_float:  TRUE
-    theme:      "lumen"
-    number_sections: FALSE
-    code_folding: "hide"
-    self_contained: TRUE
-    includes:
-      in_header: pigx_bsseq_logo.html
+title: "PiGx-BSseq: Sample Report"
+author: "BIMSB Bioinformatics Platform"
 date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
-bibliography: reports.bib
 link-citations: yes
+output:
+  html_notebook:
+      toc : TRUE
+      depth : 2
+      toc_float : TRUE
+      theme : 'lumen'
+      number_sections : TRUE
+      code_folding : "hide"
+      self_contained : TRUE
+      bibliography : 'report_templates/reports.bib'
 params:
-  Samplename: ''
-  chrom_seqlengths: ''
+  sampleid: ''
   source_dir: ''
   out_dir:    ''
   bigwigFile: '' 
   inBam:    ''
   assembly: ''
+  context: ''
   mincov:   10
   minqual:  20
   TSS_plotlength: 5000
-  methCallRDS: ''
-  methSegGR: ''
+  methCall_tabixfile: ''
   methSegBed: ''
   methSegPng: ''
-  genome_dir: ''
   scripts_dir: ''
   refGenes_bedfile:   ''
   webfetch:     ''
-  methylDiff_files: list()
+  
+  prefix:                 ''
+  workdir:                ''
+  logo:                   ''
+  
   MethCall: TRUE
   AnnotateDiffMeth: TRUE
   AnnotateSegments: TRUE
   DiffMeth: TRUE
   Segmentation: TRUE
 ---
 
+<style>
+#logo
+{
+    position: relative;
+}
+#logo img {
+    /*position: relative;*/
+    top: 25px;
+    /*right: 0px;*/
+    left: 50px;
+    position: fixed;
```

Comment:
Is fixed positioning really what you want here?

Why include the commented properties above?